test/xpu_api/tuple/tuple.tuple/tuple.cons/constexpr2_g.pass.cpp

```diff
@@ -0,0 +1,127 @@
+// -*- C++ -*-
```

Comment:
Could you please check if there is a valueable difference between this test and `constexpr1_g.pass.cpp`? If there is no difference, let's remove `constexpr2_g.pass.cpp`.

Reply:
Some difference really exists.

Reply:
I have checked one more time and it seems like the only difference is number of elements in the tuple. I guess `constexpr2_g.pass.cpp` can be removed as duplicated.